src/app/app.component.ts

```diff
@@ -2,23 +2,53 @@ import { Component } from '@angular/core';
 import { TranslateService } from '@ngx-translate/core';
 import { Store } from '@ngrx/store';
 import { login } from './components/login/auth.actions';
+import {
+  Router,
+  NavigationStart,
+  NavigationEnd,
+  NavigationCancel,
+  NavigationError,
+  RouterEvent,
+} from '@angular/router';
+import { RecipeEntityService } from './components/recipes/recipes-entity.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss'],
 })
 export class AppComponent {
-  constructor(private translate: TranslateService, private store: Store) {
+  constructor(
+    private translate: TranslateService,
+    private store: Store,
+    private router: Router,
+    private recipeService: RecipeEntityService
+  ) {
+    this.router.events.subscribe((e: RouterEvent) => {
```

Comment:
I don't like single letter variables, but I can imagine this coming from Angular docs.